integrations/helm/chartsync/chartsync.go

```diff
@@ -38,22 +36,27 @@ type Polling struct {
 	Timeout  time.Duration
 }
 
+type Clients struct {
+	KubeClient kubernetes.Clientset
+	IfClient   ifclientset.Clientset
+}
+
 type ChartChangeSync struct {
 	logger log.Logger
 	Polling
 	kubeClient          kubernetes.Clientset
```

Comment:
nit: We created a new struct called `Clients` above. Should we include this in the `ChartChangeSync` like we do with `Polling` instead of redefining and expanding the types?